lib/libvsc/VSC_cond.vsc

```diff
@@ -0,0 +1,47 @@
+..
+	This is *NOT* a RST file but the syntax has been chosen so
+	that it may become an RST file at some later date.
+
+.. varnish_vsc_begin::	cond
+	:oneliner:	Condition Counters
+	:order:		70
+
+	Counters which track the activity in the different classes
+	of condition variables.
+
+.. varnish_vsc:: init
+	:type:	counter
+	:level:	debug
+	:oneliner:	Initialized condition variables.
+
+.. varnish_vsc:: fini
+	:type:	counter
+	:level:	debug
+	:oneliner:	Uninitialized condition variables.
+
+.. varnish_vsc:: signal
+	:type:	counter
+	:level:	debug
+	:oneliner:	Number of times a signal is posted on condition variables.
+
+.. varnish_vsc:: broadcast
+	:type:	counter
+	:level:	debug
+	:oneliner:	Number of times a broadcast is posted on condition variables.
+
+.. varnish_vsc:: waits
+	:type:	counter
+	:level:	debug
+	:oneliner:	Number of times a wait occurred on condition variables.
+
+.. varnish_vsc:: waiting
+    :type:	gauge
+	:level:	debug
+	:oneliner:	Number of currently waiting thread on condition variables.
+
+.. varnish_vsc:: wait_duration_ns
+	:type:	counter
+	:level:	debug
+	:oneliner:	Amount of nanoseconds spent waiting of condition variables.
```

Comment:
fyi, an attempt at this was rejected last time around: https://github.com/varnishcache/varnish-cache/pull/3977
Not sure if adding this to non-core locks is OK.

Reply:
This amount of time is an amount of time spent waiting on a condition, it is not related to locks per se. I would therefore argue that it is not the same. Isn't the change you mention measures the amount of time in a critical section ?

Reply:
It was both, time spent waiting for the lock or condvar.
As I said, it could be OK outside of the core. I just mentioned it as it was closely related.

Reply:
alright :+1: